docs/docs/dev_docs/getting_started/cli.md

```diff
@@ -4,40 +4,50 @@ title: Aztec CLI
 
 ## Introduction
 
-The Aztec CLI is a tool designed to enable a user to interact with the Aztec Network. Here we will provide a walk-through demonstrating how to use the CLI to deploy and test contracts on the [Aztec Sandbox](../sandbox/main.md).
+The Aztec CLI is a tool designed to enable a user to interact with the Aztec Network.
+
+This tutorial will use the Aztec Sandbox so you should first set it up by following [the Aztec Sandbox instructions](../sandbox/main.md).
 
 ## Requirements
 
-The Aztec CLI is an npm package so you will need to have installed node.js >= 18. This tutorial will use the Aztec Sandbox so you should first set up the sandbox using the link above.
+You should also have [node.js](https://nodejs.org/en/download) installed with version >= 18.
 
 To install the Aztec CLI run:
 
-`npm install -g @aztec/cli`
+```bash
+npm install -g @aztec/cli
+```
 
 Or if you use yarn:
 
-`yarn global add @aztec/cli`
+```bash
+yarn global add @aztec/cli
+```
```

Comment:
Not required for this PR, but we should eventually use tabs for expressing the same command across multiple languages / systems, because they're cool: https://docusaurus.io/docs/markdown-features/code-blocks#multi-language-support-code-blocks